src/libraries/System.IO.Compression.ZipFile/src/System.IO.Compression.ZipFile.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
```

This makes sure that the Unix is picked as the default.

```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
```

---

Shouldn't we also remove '$(TargetPlatformIdentifier)' == 'Unix'` if we make this change

---

Ok, thanks pushing the changes.